code/pages/rss.fs

```diff
@@ -0,0 +1,22 @@
+﻿module FsSnip.Pages.Rss
+
+open Suave
+open Suave.Http
+open System
+open FsSnip.Utils
+open FsSnip.Data
+open FsSnip.Rssfeed
+open FsSnip.Filters
+
+let getRecent () =
+  publicSnippets
+  |> Seq.sortBy (fun s -> DateTime.Now - s.Date)
+  |> Seq.take 10
+
+let getRss = fun (ctx: Types.HttpContext) -> async {
```

Comment:
There is a helper function in `utils.fs` that lets you write this as:

    let getRss = delay (fun () -> 
        let rssOutput = ...
        Successful.OK rssOutput)

This might be a bit nicer here because we're not doing anything asynchronously.